samples/unit-tests/series-heatmap/members/demo.js

```diff
@@ -219,3 +219,46 @@ QUnit.test('seriesTypes.heatmap.pointClass.setState', function (assert) {
         'When state:select zIndex is 0'
     );
 });
+
+QUnit.test('Check if borderRadius is correct according to user settings', function (assert) {
```

Comment:
Usually, we don't want to create a separate test with a new chart's instance, because creating thousands of new charts during the tests increases theirs time. Can we reuse the above test that tests something very similar?

Reply:
Sure 👍